HTML/QuickForm/header.php

```diff
@@ -60,10 +60,12 @@ function __construct($elementName = null, $text = null)
     * Accepts a renderer
     *
     * @param HTML_QuickForm_Renderer    renderer object
+    * @param bool $sc1                  unused, for signature compatibility
+    * @param bool $sc2                  unused, for signature compatibility
     * @access public
     * @return void
     */
-    function accept(&$renderer)
+    function accept(&$renderer, $sc1 = false, $sc2 = null)
```

Comment:
change 8

Reply:
Also not matching the parent
```
    * @param HTML_QuickForm_Renderer    renderer object
    * @param bool                       Whether an element is required
    * @param string                     An error message associated with an element
    * @access public
    * @return void
    */
    function accept(&$renderer, $required=false, $error=null)
    {
        $renderer->renderElement($this, $required, $error);
    } // end func accept

    // }}}
```

Reply:
Again change comes from straight upstream https://gist.github.com/seamuslee001/0a67ad08359f46b183a7e26f05c99658#file-quickform_diff-diff-L812